### DISCUSSION TITLE
Is there anything I should have enabled that is disabled when using "https://ffprofile.com/" such that this works with my current browser profile?

### DISCUSSION THREAD

**Discussion by FFFluoride** [dyc3/opentogethertube#1773] [Q&A] [answered]
I used https://ffprofile.com/ to change my firefox browsing experience. This (probably) causes the "https://opentogethertube.com/" page not to load as this is what I have observed. It loads with a fresh profile so it's definitely my profile that is the issue.

If anyone has any insight as to what specifically might be causing this, I would love to know.

**Comment by dyc3** [comment]
Just to make sure, does it work in another browser?

**Reply by FFFluoride** [reply]
Yes, it works in stock firefox and stock chromium (at least)

**Reply by dyc3** [reply]
Could you post a screenshot of what you see in the network tab of the dev tools when you try to load the page?

**Reply by FFFluoride** [reply]
Sure thing!

![Screenshot from 2024-05-08 15-25-31](https://github.com/dyc3/opentogethertube/assets/82343884/d9e8b690-5d3d-4548-b07a-242f59d2a18b)

**Reply by dyc3** [reply, marked as answer]
You'll need to have "Website Tracking > Disable webaudio API" unchecked.

**Reply by FFFluoride** [reply]
Thanks man
